apps/demo-app/src/styles/modules/theming/_theming.scss

```diff
@@ -1,56 +1,97 @@
+/* stylelint-disable number-max-precision */
 @use '../variables' as vars;
 @use 'sass:list';
 
 $theme: (
-	header: (
-		header: (
-			background: vars.$primary-lightest,
-			title: (
-				size: 32px,
-				color: vars.$primary,
-			),
+	general: (
+		colors: (
+			primary: #2581C4,
+			accent: #EA518D,
+			dark: #323232,
+			disabled: #D9D9D9,
+			disabled-light: #F5F5F5
+		),
+		font: (
+			family: '',
+			size: 14px
 		),
-		button-primary: (
-			default-background: vars.$primary,
-			default-font-color: vars.$font-color-light,
-			hover-background: vars.$accent,
-			hover-font-color: vars.$font-color-light,
-		),
-		button-accent: (
-			default-background: vars.$accent,
-			default-font-color: vars.$font-color-light,
-			hover-background: vars.$primary,
-			hover-font-color: vars.$font-color-light,
+		buttons: (
+			color: #FFF
+		),
+		inputs: (
+			background-color: #FFF
 		)
 	),
-	dialog: (
-		header: (
-			title: (
-				size: 20px,
-				color: vars.$font-color-default,
-			),
-			close-icon: (
-				size: 12px,
-				default-color: vars.$font-color-default,
-				hover-color: vars.$primary,
+	forms: (
+		fields: (
+			infix-padding-top: .4375em,
```

Comment:
@Cvetanovic  Is it possible to circle this to rounded number and replace it with px units?